app/jobs/dttp/sync_users_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Dttp
+  class SyncUsersJob < ApplicationJob
+    queue_as :dttp
+
+    def perform(path = nil)
+      user_list = Dttp::RetrieveUsers.call(path: path)
+
+      user_list[:items].each do |user_hash|
```

Comment:
Could we use `upsert_all` here and do the whole page in one go?

Reply:
done